operator/pom.xml

```diff
@@ -68,6 +65,8 @@
         <dependency>
             <groupId>io.quarkiverse.operatorsdk</groupId>
             <artifactId>quarkus-operator-sdk-bundle-generator</artifactId>
+            <scope>compile</scope>
+            <optional>true</optional>
         </dependency>
```

Comment:
The bundle generator seems to be still present in the resulting ZIP. @shawkins Can you please check it? I assume we can't scope it as build time only, right?

Reply:
As provided it will no longer create the bundle output and the olm assumbly will fail.  It looks like you have to explicitly turn on optional dependency filtering in quarkus.